### DISCUSSION TITLE
juicefs unknown option: --no-posix-lock

### DISCUSSION THREAD

**Discussion by ottacom** [juicedata/juicefs#3598] [General]
**What you expected to happen**:
Mounting  the file system with the option --no-posix-lock by the link 
https://juicefs.com/docs/cloud/reference/commands_reference/#fuse-options
**How to reproduce it (as minimally and precisely as possible)**:
juicefs mount -d redis://localhost:6379/1 --no-posix-lock  /var/lib/libvirt/images
**Anything else we need to know?**
n.a.
**Environment**:
- JuiceFS version 1.0.4+2023-04-06.f1c475d
- Cloud provider or hardware configuration running JuiceFS:
- OS Ubuntu 22
- Kernel Linux 5.15.0-71-generic 78-Ubuntu SMP Tue Apr 18 09:00:29 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux
- Object storage ( self maintained):
- Metadata engine info ( self maintained):
- Network connectivity ( JuiceFS to object storage):
- Others:

**Comment by davies** [comment]
`--no-posix-lock` is an option for cloud edition, not for community edition. Here is the docs for community edition: https://juicefs.com/docs/community/command_reference
